.gitignore

```diff
@@ -1 +1 @@
-node_modules/
+node_modules
```

Comment:
Sure, I didn't even notice the line wasn't the same. I'm actually going to cherry pick the main commit into a new branch and create a new pull request so we won't even see the .gitignore commits in master.